docs/useGetTree.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: "useGetTree"
+---
+
+# `useGetTree`
+
+This [Enterprise Edition](https://marmelab.com/ra-enterprise)<img class="icon" src="./img/premium.svg" /> hook is exposed by [the `ra-tree` module](https://marmelab.com/ra-enterprise/modules/ra-tree). It calls `dataProvider.getTree()` when the component mounts. It's ideal for getting nodes of the tree structure.
+
+![useGetTree](https://marmelab.com/ra-enterprise/modules/assets/ra-tree-overview.gif)
+
+## Syntax
+
+```jsx
+const { data, error, isLoading, isFetching } = useGetTree(
+    resource,
+    { meta },
+    options
+);
+```
+
+## Usage
+
+```jsx
+import { useGetTree } from '@react-admin/ra-tree';
+const Categories = () => {
+    const { data, isLoading, error } = useGetTree('categories');
+    if (isLoading) { return <Loading />; }
+    if (error) { return <p>ERROR</p>; }
+    return <Tree tree={data} />;
+};
+```
```

Comment:
You should complete this code example with exhaustive imports.
At least to show that the `Tree` component is also provided by ra-tree.